.vscode/settings.json

```
@@ -30,5 +30,8 @@
     },
     "clangd.arguments": [
         "--header-insertion=never"
-    ]
+    ],
+    "cmake.configureOnOpen": true,
+    "C_Cpp.errorSquiggles": "disabled",
+    "cmake.ignoreCMakeListsMissing": true
```

This file needs to be reverted
```suggestion
    ]
```